scripts/actions/abilities/reward.lua

```diff
@@ -16,7 +16,10 @@ abilityObject.onAbilityCheck = function(player, target, ability)
         return xi.msg.basic.NO_EFFECT_ON_PET, 0
     else
         local id = player:getEquipID(xi.slot.AMMO)
-        if id >= 17016 and id <= 17023 then
+        if
+            id >= 17016 and
+            id <= 17023
+        then
```

Comment:
While you are in this very file...

Could you do the same to line 15 and add those item IDs to the enum file?

Reply:
Sure thing. Done.